examples/ssd/caffe2npz.py

```diff
@@ -59,17 +61,18 @@ def add_link(self, name, link):
                 data = data.reshape((-1, 4) + data.shape[1:])
                 data[:, [1, 0, 3, 2]] = data.copy()
             print('{:s} -> {:s} (xy -> yx)'.format(name, new_name))
-        else:
+        elif match:
```

Comment:
Could you remove `match`? It is better to show all layers even if their names are not changed. And this modification is not related to supporting v2 and v3.